script/component-migration/transformers/additionalinfo.js

```diff
@@ -1,15 +1,18 @@
 function additionalInfoReplacement() {
-  const removeClassName = (componentString, _propName) => {
-    let noClass = componentString;
-
-    noClass = noClass.replace(/className=["{](.)+?["}]\s/ms, '');
-    return noClass;
+  const removeProp = (componentString, propName, newTag) => {
+    const noProp = componentString;
+    const regex = new RegExp(
+      `(?<=<${newTag}.+?)(${propName}=["{].+?[}"])(?=.+?>)`,
+      's',
+    );
+    noProp.replace(regex, '');
+    return noProp;
```

Comment:
I don't think using `newTag` like this will work. Here's a [link to a regex playground](regexr.com/6bhul) where I've copied this regex and a sample component, and it's still matching on `className` props in child nodes:

![image](https://user-images.githubusercontent.com/2008881/146039931-bab62427-9d53-4f94-a432-f2ea7b8e149e.png)